assets/src/components/activities/common/choices/delivery/ChoicesDelivery.tsx

```diff
@@ -24,13 +24,27 @@ export const ChoicesDelivery: React.FC<Props> = ({
   selectedIcon,
 }) => {
   const isSelected = (choiceId: ChoiceId) => !!selected.find((s) => s === choiceId);
+
+  const onClicked = useCallback(
+    (choiceId: ChoiceId) => (event: React.MouseEvent) => {
+      if (event.isDefaultPrevented()) {
```

Comment:
Went with a preventDefault solution over stopPropagation to be a little more light-handed to let other components decide how to handle events.